java/vector/src/test/java/org/apache/arrow/vector/compare/TestRangeEqualsVisitor.java

```diff
@@ -434,25 +434,39 @@ public void testStructVectorRangeEquals() {
       NullableStructWriter writer1 = vector1.getWriter();
       writer1.allocate();
 
+      writeStructVector(writer1, 0, 0L);
       writeStructVector(writer1, 1, 10L);
       writeStructVector(writer1, 2, 20L);
       writeStructVector(writer1, 3, 30L);
       writeStructVector(writer1, 4, 40L);
       writeStructVector(writer1, 5, 50L);
-      writer1.setValueCount(5);
+      writer1.setValueCount(6);
```

Comment:
We are adding 6 values? And why we set count to 5?

Reply:
I just want to test the case where the left&right index in the range are different.
This is better than the original case where the left & right index are identical.